.github/workflows/check.yml

```diff
@@ -0,0 +1,50 @@
+name: check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  print_hip_info:
+    strategy:
+      fail-fast: false
+      matrix:
+        toolchain: [gcc12]
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: bazelbuild/setup-bazelisk@v2
```

Comment:
github runners come with bazelisk, no need to run this action (as noted in https://github.com/bazelbuild/setup-bazelisk/blob/main/README.md)

Reply:
done